beacon-chain/state/state-native/beacon_state_mainnet.go

```diff
@@ -103,8 +115,19 @@ type beaconStateMarshalable struct {
 	NextSyncCommittee                   *ethpb.SyncCommittee                    `json:"next_sync_committee" yaml:"next_sync_committee"`
 	LatestExecutionPayloadHeader        *enginev1.ExecutionPayloadHeader        `json:"latest_execution_payload_header" yaml:"latest_execution_payload_header"`
 	LatestExecutionPayloadHeaderCapella *enginev1.ExecutionPayloadHeaderCapella `json:"latest_execution_payload_header_capella" yaml:"latest_execution_payload_header_capella"`
+	LatestExecutionPayloadHeaderDeneb   *enginev1.ExecutionPayloadHeaderDeneb   `json:"latest_execution_payload_header_deneb" yaml:"latest_execution_payload_header_deneb"`
```

Comment:
I wonder what is breaking today we don't have deneb here

Reply:
hmmm yeah good point, i dont know lol

Reply:
Oh, this is to json-ify the beacon state, which I am guessing we never / rarely do because no one noticed this field was missing in the JSON version of the mainnet beacon state in deneb